sigs.yaml

```diff
@@ -201,6 +228,63 @@ sigs:
         teams:
         - name: kubevirt-community
           description: General Discussion
+      subprojects:
+      - name: ci-s390x
+        description: |
+          The ci-s390x subproject takes care of all matters related to CI for s390x architecture.
+          leads:
+          - github: jschintag
+            name: "Jan Schintag"
+            company: IBM
+          - github: cfilleke
+            name: "Cheryl Fillekes"
+            company: IBM
+          - github: vamsikrishna-siddu
+            name: "vamsikrishna-siddu"
+            company: IBM
+          - github: chandramerla
+            name: "Chandra Merla"
+            company: IBM
+        owners:
+        - https://raw.githubusercontent.com/kubevirt/project-infra/main/OWNERS
+      - name: ci-arm
+        description: |
+          The ciarch-arm subproject takes care of all matters related to CI for arm architecture.
+        leads:
+        - github: zhlhahaha
+          name: "Howard Zhang"
+          company: ARM
+        owners:
+        - https://raw.githubusercontent.com/kubevirt/project-infra/main/OWNERS
+workinggroups:
+    - dir: wg-arch-s390x
+      name: S390x architecture working group
+      mission_statement: |
+        WG ARCH s390x takes care of all things related to supporting s390x architecture on KubeVirt clusters.
+        This includes all build processes, also support of failing builds or test lanes.
+      label: wg-arch-s390x
+      leadership:
+        chairs:
+        - github: jschintag
+          name: "Jan Schintag"
+          company: IBM
+        - github: cfilleke
```

Comment:
@cfilleke - I can't see you as a member in the KubeVirt org. Can you please open a PR in the project-infra repo and add your name to the [orgs.yaml](https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml): github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml

You meet [the criteria](https://github.com/kubevirt/community/blob/main/membership_policy.md) so this is just the formality. I would be happy to sponsor you.

Reply:
+1 - so would I!